includes/admin/views/html-notice-base-table-missing.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Admin View: Notice - Base table missing.
+ *
+ * @package WooCommerce\Admin
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+?>
+<div class="updated woocommerce-message">
+	<a class="woocommerce-message-close notice-dismiss"
+	   href="<?php echo esc_url( wp_nonce_url( add_query_arg( 'wc-hide-notice', 'base_tables_missing' ), 'woocommerce_hide_notices_nonce', '_wc_notice_nonce' ) ); ?>">
+			<?php esc_html_e( 'Dismiss', 'woocommerce' ); ?>
+	</a>
+
+	<p>
+		<strong><?php esc_html_e( 'Database tables missing', 'woocommerce' ); ?></strong>
+	</p>
+	<p>
+		<?php
+		$verify_db_tool_available = array_key_exists( 'verify_db_tables', WC_Admin_Status::get_tools() );
+		$missing_tables           = get_option( 'woocommerce-schema-missing-tables' );
```

Comment:
It need to get updated to `woocommerce-schema-missing-tables`, by default on WooCommerce we use `woocommerce_` as a prefix for all options, note that also we remove options with this prefix when running the `unistall.php`.

Reply:
Changed in https://github.com/woocommerce/woocommerce/pull/26454/commits/4fce106f1446eb5e8fec45bd040b2cacde971880